data.json

```diff
@@ -5934,7 +5951,9 @@
                 "2EF4_wordpad.0",
                 "97C1_wordpad.0",
                 "applications-office",
-                "wordpad"
+                "wordpad",
+                "wordview",
+                "wordview.xpm"
```

Comment:
That's one to be fixed by a hardcode fixer, not by us adding a symlink on our end.

Reply:
By the way, this isn't just a nod to our workflow, adding a double file extension just doesn't work, the icon still won't be shown.